jsonHandler.py

```diff
@@ -0,0 +1,145 @@
+import json
+import datetime
+from events import *
+
+class JsonHandler(object):
+
+    def __init__(self):
+        self._filename = "json_data.json"
+
+
+    def add_events(self,event):
+        data = None
+        with open(self._filename,'r') as data_file:
+            data = json.load(data_file)
+        d={event.get_event_id():event.toJson()}
+        data.update(d)
+        with open(self._filename,'w+') as changed:
+            changed.write(json.dumps(data,indent=4))
+
+
+    def delete_event(self,event_id):
+        data = None
+        with open(self._filename, 'r') as data_file:
+            data = json.load(data_file)
+        del data[event_id]
+        with open(self._filename,'w+') as changed:
+            changed.write(json.dumps(data,indent=4))
+
+
+    def update_event(self,event):
+        data = None
+        with open(self._filename,'r') as data_file:
+            data = json.load(data_file)
+        d=event.toJson()
+        data[event.get_event_id()]=d
+        with open(self._filename,'w+') as changed:
+            changed.write(json.dumps(data,indent=4))
+
+
+    def read_event(self,event_id):
+        data = None
+        with open(self._filename,'r') as data_file:
+            data = json.load(data_file)
+        event=Event()
+        event.set_event_id(event_id)
+        event.set_name(data[event_id]['name'])
+        event.set_info(data[event_id]['info'])
+        event.set_date(data[event_id]['date'])
+        event.set_venue(data[event_id]['venue'])
+        event.set_city(data[event_id]['city'])
+        return event
```

Comment:
instead, u can create a constructor in Event class and pass and initialize all of them in one go